src/libmeasurement_kit/ooni/ooni_test.cpp

```diff
@@ -77,30 +78,27 @@ void OoniTest::geoip_lookup(Callback<> cb) {
                     logger->debug("saving user's real ip on user's request");
                     probe_ip = ip;
                 }
-                std::string country_p =
-                    options.get("geoip_country_path", std::string{});
-                std::string asn_p =
-                    options.get("geoip_asn_path", std::string{});
-                if (country_p == "" or asn_p == "") {
+                auto cntry_p = options.get("geoip_country_path", std::string{});
+                auto asn_p = options.get("geoip_asn_path", std::string{});
+                if (cntry_p == "" or asn_p == "") {
```

Comment:
Maybe we can check only for the paths we actually need. That is `cntry_p` if `save_real_probe_country` is true and `asn_p` if `save_real_probe_asn` is true.